index.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 const events = require('events')
-const x = require('0x')
+const x = require('https://github.com/DylanC/0x/tree/collect_timeout_feature')
```

Comment:
that doesn't quite work :sweat_smile: node only loads dependencies from the file system.

You can tell npm to install 0x from a different location by doing this in the package.json file:
```json
"dependencies": {
  "0x": "DylanC/0x#collect_timeout_feature"
}
```

Reply:
@goto-bus-stop - Ah makes sense. I need some more coffee this morning. ☕️